lib/auth/touchid/attempt.go

```diff
@@ -64,3 +64,21 @@ func AttemptLogin(origin, user string, assertion *wanlib.CredentialAssertion, pi
 	}
 	return resp, actualUser, nil
 }
+
+// AttemptDeleteNonInteractive attempts to delete a Secure Enclave credential.
+// Does not require user interaction.
+func AttemptDeleteNonInteractive(credentialID string) error {
+	if !IsAvailable() {
+		return &ErrAttemptFailed{Err: ErrNotAvailable}
+	}
+	if credentialID == "" {
+		return errors.New("credentialID required")
+	}
+	if err := native.DeleteNonInteractive(credentialID); err != nil {
+		if errors.Is(err, ErrCredentialNotFound) {
+			return &ErrAttemptFailed{Err: err}
+		}
+		return trace.Wrap(err)
+	}
```

Comment:
Suggestion: a bit less nesting? Either way is fine, obviously.

```suggestion
	switch err := native.DeleteNonInteractive(credentialID); {
	case errors.Is(err, ErrCredentialNotFound):
		return &ErrAttemptFailed{Err: err}
	case err != nil:
		return trace.Wrap(err)
	}
```

Reply:
applied